addons/sys_gui/CfgRscTitles.hpp

```diff
@@ -4,6 +4,72 @@ class RscXSliderH;
 class RscText;
 class RscStructuredText;
 class RscControlsGroupNoScrollbars;
+class RscPicture;
+class RscPictureKeepAspect;
+
+class RscIngameUI {
+    class RscStanceInfo {
+        idd=303;
+        scriptName="RscStanceInfo";
+        scriptPath="IGUI";
+        onLoad="[""onLoad"",_this,""RscStanceInfo"",'IGUI'] call    (uinamespace getvariable 'BIS_fnc_initDisplay')";
```

Comment:
- `uiNamespace`
- `getVariable`
- Spaces around `=`
- Space after `,`
- Excessive spacing.

This and next file.